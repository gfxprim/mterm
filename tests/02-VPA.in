2 3
\e[8da
