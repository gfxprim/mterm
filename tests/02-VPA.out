 --
|  |
|  |
|a |
 --
size 3x2 cursor 2x1
